metrics.go

```diff
@@ -64,6 +64,16 @@ func (sender *Sender) Increment(bucket string) {
 	}
 }
 
+// Gauge sends a gauge metric
+func Gauge(bucket string, value interface{}) { sender.Gauge(bucket, value) }
+
+// Gauge sends a gauge metric
+func (sender *Sender) Gauge(bucket string, value interface{}) {
```

Comment:
Why no "`default`" case? I suppose that the type list is complete but seeing a switch without a default makes me uneasy, especially with a type signature as wide as "`interface {}`".